### PR TITLE
Update shares on group removal

### PR DESCRIPTION
**Pull request by artonge**

### CODE REVIEW

File: apps/files_sharing/lib/Listener/UserRemoveFromGroupListener.php [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+use OCP\EventDispatcher\IEventListener;
+use OCP\Group\Events\UserRemovedEvent;
+use OCP\IConfig;
+use OCP\Share\IManager;
+
+/** @template-implements IEventListener<UserRemovedEvent> */
+class UserRemoveFromGroupListener implements IEventListener {
+	public function __construct(
+		private IManager $shareManager,
+		private IConfig $config,
```
Comment (susnux) [inline apps/files_sharing/lib/Listener/UserRemoveFromGroupListener.php:38]: Unused?

Comment (artonge) [reply apps/files_sharing/lib/Listener/UserRemoveFromGroupListener.php:38]: Indeed

**Review by susnux — APPROVED** [review]
Looks good to me, we should have integration tests for this :no_mouth:

**Review by Altahrim — APPROVED** [review]

**Comment by artonge** [comment]
False positive, a listener was already added: https://github.com/nextcloud/server/blob/8b768210d2ac28b236617b20c19498085ddcf328/lib/private/Share20/UserRemovedListener.php

**Pull request closed without merging**